client/packages/electron/src/electron.ts

```diff
@@ -1,43 +1,21 @@
 import { app, BrowserWindow, ipcMain } from 'electron';
-import startDiscovery from 'multicast-dns';
+import dnssd from 'dnssd';
 import { IPC_MESSAGES } from './shared';
-import ip from 'ip';
+import thisIp from 'ip';
 import {
   FrontEndHost,
   frontEndHostUrl,
-} from '@openmsupply-client/common/src/hooks/useElectronClient';
-import storage from 'electron-data-storage';
-
-const QUERY_INTERVAL = 1000;
-const SERVICE_NAME = '_omsupply._tcp.local';
-const SERVICE_TYPE = 'TXT';
-
-class DiscoveryRunner {
-  intervalHandle?: NodeJS.Timer;
-
-  start() {
-    this.intervalHandle = setInterval(() => {
-      discovery.query({
-        questions: [
-          {
-            name: SERVICE_NAME,
-            type: SERVICE_TYPE,
-          },
-        ],
-      });
-    }, QUERY_INTERVAL);
-  }
+  isProtocol,
+} from '@openmsupply-client/common/src/hooks/useNativeClient';
+const SERVICE_TYPE = 'omsupply';
+const PROTOCOL_KEY = 'protocol';
+const CLIENT_VERSION_KEY = 'client_version';
+const HARDWARE_ID_KEY = 'hardware_id';
 
-  stop() {
-    if (this.intervalHandle) clearInterval(this.intervalHandle);
-    this.intervalHandle = undefined;
-  }
-}
-
-const discovery = startDiscovery();
-const discoverRunner = new DiscoveryRunner();
+const discovery = new dnssd.Browser(dnssd.tcp(SERVICE_TYPE));
```

Comment:
This makes this tidier I think (and can stop/start/restart with more confidence)